src/renderer/redirect.ts

```diff
@@ -1,23 +1,16 @@
 import { ActionResult } from '../models/response.ts';
 
-export function Redirect(url: string): ActionResult {
+export function Redirect(url: string, permanent: boolean = false): ActionResult {
     const headers = new Headers();
     headers.append("Location", url);
 
     return {
-        status: 302,
+        status: !permanent ? 302 : 301,
         headers,
         __isActionResult: true,
     };
 }
 
-export function RedirectPermanent(url: string): ActionResult {
-    const headers = new Headers();
-    headers.append("Location", url);
-    
-    return {
-        status: 301,
-        headers,
-        __isActionResult: true,
-      };
-}    
\ No newline at end of file
+export function PermanentRedirect(url: string) {
```

Comment:
like this https://docs.microsoft.com/en-gb/dotnet/api/system.web.httpresponse.redirectpermanent?view=netframework-4.8